__tests__/app.test.js

```diff
@@ -158,6 +159,49 @@ describe("GET /api/articles", () => {
         });
       });
   });
+  test("Status:200 default sort_by criteria: created_at and default ORDER BY criteria: DESC descending", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+  test("Status:200 sort by the passed query", () => {
+    return request(app)
+      .get("/api/articles?sort_by=title")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("title", { descending: true });
+      });
+  });
+  test("Status:200 order by the passed query", () => {
+    return request(app)
+      .get("/api/articles?order_by=ASC")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("created_at", { descending: false });
+      });
+  });
+  test("Status:200 filter topic by the passed query", () => {
+    return request(app)
+      .get("/api/articles?filter_topic_by=cats")
```

Comment:
I suggest instead of having a long query variable of `filter_topic_by`, I would just call it `topic` for ease!

Reply:
thanks for the advice :)